default.xml

```diff
@@ -26,15 +26,24 @@
   <project path="packages/apps/Launcher3" name="platform/packages/apps/Launcher3" groups="pdk-fs" remote="aosp" />
   <project path="packages/apps/LineageParts" name="android_packages_apps_LineageParts" remote="maruos" />
   <project path="prebuilts/busybox" name="platform_prebuilts_busybox" remote="maruos" revision="master" />
-
+  <project name="LineageOS/android_packages_resources_devicesettings" path="packages/resources/devicesettings" remote="github" revision="lineage-16.0" />
```

Comment:
Nit: could you move it to the top of maruos part?

Reply:
I could but that's not really a maruos thing. You would hit the same problem and require this repo while building classic LOS.

Where do you suggest to put it ?
above lxc or above vendor/maruos ?

Reply:
Maybe.

Reply:
Hmmm, it's not a huge deal of course but it would be nice if `default.xml` was purely Maru repos. 

Here are some options:

1. The only other file we have where we specify customizations is in `snippets/overrides.xml`. At the bottom of that file we remove some unnecessary LOS repos for example. So it's possible it could go in there but I guess this isn't really an "override".
2. We could possibly create another file in the `snippets` directory called `extras.xml` or something and add it in there.
3. We can just move it to either above or below all the `maruos` repos in `default.xml`.

Any opinions which would be best?

Reply:
I prefer to choice 3, because we can only maintain one file for manifest. And the next is choice 2, we can include it with `include` tag.

Reply:
I prefer choice 3 as well. @pintaf, can you move this above all the `maruos` repos in our manifest?

Reply:
I also think choice 3 is the best. However instead of above, I think bellow the maruos repos would be better.
Indeed, this repo need came with my HTC10, but it is also needed by some other devices. Hence it is mostly device related and should be closer between maru repos and devices repos in my opinion.

I'll integrate my HTC repos in maruos and update this PR. Maybe tomorrow, maybe Thursday, I'm sinking under the things to do.

Reply:
OK below is fine with me especially since it is device-related. Take your time on the PR, no rush.